update.sh

```diff
@@ -35,7 +35,61 @@ InstallWineRunner() {
   curl -sL "$url" | tar xfzv - -C "$dest_path"
   install_complete=true
   echo "Installation completed"
-  RestartLutrisCheck
+  Wannadelete
+}
+
+Wannadelete() {
+    if [ "$( pgrep lutris )" != "" ] && [ "$install_complete" = true ]; then
+        read -r -p "Do you want to delete intalled runners? <y/N> " prompt
+        if [[ $prompt == "y" || $prompt == "Y" || $prompt == "yes" || $prompt == "Yes" ]]; then
+            DeleteRunnersCheck
+        else
+            RestartLutrisCheck
+        fi
+    fi
+}
+
+DeleteRunnersCheck() {
+    echo "Installed runners:"
+    installed_versions=($(ls -d "$base_path"/*/))
+    for((i=0;i<${#installed_versions[@]};i++)); do
+        inumber=$(("$i" + 1))
+        folder=$(echo "${installed_versions[i]}" | rev | cut -d/ -f2 | rev)
+        echo "$inumber. $folder"
+    done
+    echo -n "Please choose an option to remove [1-${#installed_versions[@]}]:"
```

Comment:
Can we add an empty line here to keep styling similar to the other `choose option` function?

```
echo ""
echo -n "Please choose an option to remove [1-${#installed_versions[@]}]:"
```

Reply:
Didn't catch that, thanks! :)